features/270-next-skycli/commands.md

```diff
@@ -64,26 +66,35 @@ View the current config.
 #### Example
 ```
 $ skycli config view
-[cluster]
-email=user@skygear.io
-cluster-server=https://1.2.3.4
-admin-key=
-
-[app]
-app=myapp
+api_version: v1
+clusters:
+  - name: skygeario
+    cluster:
+      env: cloud
+      endpoint: 'https://controller.example.com'
+      api_key: api-key
+users: []
+contexts:
+  - name: skygeario
+    context:
+      cluster: skygeario
+      user: skygeario
+current_context: skygeario
 ```
 
 ### skycli config set-cluster
 
 `skycli config set-cluster --cluster=[CLUSTER_NAME] --endpoint=[CLUSTER_SERVER_URL] --api-key=[API_KEY|MASTER_KEY]`
 
-Change current cluster controller server endpoint and api key, it is provided
-by cluster admin. If cluster admin want to use admin only api, they should provide
-master key in the api key flag.
+Configure Cluster endpoint and API key, it is provided by cluster admin.
+If there is no current context set, the configured Cluster would be set as the
+current context.
+If there exist a cluster with same name, it is updated with provided
+information.
 
 #### Flags
 
-- `--cluster=[CLUSTER_NAME]` Provide cluster name, currently only `skygeario` is supported.
+- `--cluster=[CLUSTER_NAME]` Provide cluster name.
```

Comment:
After this update, if user want to set their own cluster without prompt, they will need to provide `cluster` flag? If yes, please update the example below. Maybe `skycli config set-cluster --cluster=mycluster --endpoint=https://mycluster-controller --api-key=api_key` ?